test/unit/mock/profiles/bad.rb

```diff
@@ -0,0 +1,3 @@
+describe file('/tmp/foo') do
```

Comment:
Please, we are trying to keep our fixtures organized; please store your fixtures under directories with meaningful names. In what way is this "bad"? It is because it is a bare control file with no metadata (which is legal)? Or a legal though failing file control referencing a non-existant file? `resource_on_bad_platform/missing_file.rb` ?

Reply:
+1 a comment describing what we're testing would be useful.

Reply:
Happy to rename it something else. Done.